### PR TITLE
🐛 Allow Alerts components to recover when alerts are unset

### PR DESCRIPTION
**Pull request by superbuggy**
# [OSIDB-ID] [Title]

## Checklist:

- [ ] Linting passed
- [ ] Type checks passed
- [ ] Tests suite passed
- [ ] Commits consolidated
- [ ] Changelog updated
- [ ] Test cases added/updated
- [ ] Jira ticket updated

## Summary:

Alerts component errors out when new affects are added

## Changes:

[Replace with a detailed description of the changes made in this PR, including any new features, enhancements, bug fixes, or refactorings.]

## Considerations:

[Replace with any additional considerations, notes, or instructions for reviewers.]

### CODE REVIEW

**Review by tboyetteredhat — APPROVED** [review]

File: src/components/FlawAlertsSection.vue [hunk 1]
```diff
@@ -48,7 +48,7 @@ const emitExpandFocusedComponent = (parent_uuid: string) => {
 
     <div v-for="alerts,alertType in alertSet" :key="alertType">
       <div v-if="alerts.length">
-        <span v-for="alert in alerts" :key="alert.uuid">
+        <span v-for="(alert, index) in alerts" :key="alert?.uuid || `alert-${index}-${alertType}`">
```
Comment (JakubFrejlach) [inline src/components/FlawAlertsSection.vue:51]: Why is this change needed? I mean `alerts` should be always a list, so there will either be some alerts and then each alert should have its own uuid, or the list will be empty, right?

**Review by JakubFrejlach — COMMENTED** [review]
This is happening when some draft model (Eg. blank affect, initial flaw) is added?

**Comment by superbuggy** [comment]
Included in #244
